build.sbt

```diff
@@ -137,13 +137,18 @@ lazy val cli = projectMatrix
     moduleName := "scalafix-cli",
     isFullCrossVersion,
     libraryDependencies ++= Seq(
-      java8Compat,
```

Comment:
`java8Compat` dependency couldn't be found for `cli3/compile`, hence removing it